packages/charts/src/hooks/useCancelAnimationFallback.ts

```diff
@@ -0,0 +1,23 @@
+import { debounce } from '@ui5/webcomponents-react-base/dist/Utils';
+import { useRef, useState } from 'react';
+
+export const useCancelAnimationFallback = (noAnimation) => {
```

Comment:
```suggestion
export const useCancelAnimationFallback = (noAnimation: boolean) => {
```
Maybe we can add some strict types here?

Reply:
Done, also added the return type.